bitherj/src/main/java/net/bither/bitherj/PrimerjSettings.java

```diff
@@ -54,13 +54,20 @@
 
     public static final BigInteger proofOfWorkLimit = Utils.decodeCompactBits(0x1d00ffffL);
     public static final int port = 9911;
+    public static final int testNetPort = 9913;
+    public static int getPort(){
+        if(Utils.isTestNet()) return testNetPort;
+        return port;
+    }
     //    public static final int port = 8333;
     public static final long packetMagic = 0xe4e7e5e7L;
+    public static final long testNetPacketMagic = 0xfbfecbc3L;
 //    public static final long packetMagic = 0xf9beb4d9L;
 
 
     //    public static final int addressHeader = 0;
     public static final int addressHeader = 23;
+    public static final int testAddressHeader = 111;
```

Comment:
To be consistent with other names, this best to be 'testNetAddressHeader'

Reply:
Ok,